GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceSettingsPage.Designer.cs

```diff
@@ -28,570 +28,563 @@ protected override void Dispose(bool disposing)
+            tlpnlGeneral.Name = "tlpnlGeneral";
+            tlpnlGeneral.RowCount = 5;
+            tlpnlGeneral.RowStyles.Add(new RowStyle());
+            tlpnlGeneral.RowStyles.Add(new RowStyle());
+            tlpnlGeneral.RowStyles.Add(new RowStyle());
+            tlpnlGeneral.RowStyles.Add(new RowStyle());
+            tlpnlGeneral.RowStyles.Add(new RowStyle());
+            tlpnlGeneral.Size = new Size(1464, 129);
+            tlpnlGeneral.TabIndex = 0;
             // 
             // chkShowRelativeDate
             // 
-            this.chkShowRelativeDate.AutoSize = true;
-            this.tlpnlGeneral.SetColumnSpan(this.chkShowRelativeDate, 2);
-            this.chkShowRelativeDate.Dock = System.Windows.Forms.DockStyle.Fill;
-            this.chkShowRelativeDate.Location = new System.Drawing.Point(3, 3);
-            this.chkShowRelativeDate.Name = "chkShowRelativeDate";
-            this.chkShowRelativeDate.Size = new System.Drawing.Size(448, 17);
-            this.chkShowRelativeDate.TabIndex = 0;
-            this.chkShowRelativeDate.Text = "Show relative date instead of full date";
-            this.chkShowRelativeDate.UseVisualStyleBackColor = true;
+            chkShowRelativeDate.AutoSize = true;
+            tlpnlGeneral.SetColumnSpan(chkShowRelativeDate, 2);
+            chkShowRelativeDate.Dock = DockStyle.Fill;
+            chkShowRelativeDate.Location = new Point(3, 3);
+            chkShowRelativeDate.Name = "chkShowRelativeDate";
+            chkShowRelativeDate.Size = new Size(501, 19);
+            chkShowRelativeDate.TabIndex = 0;
+            chkShowRelativeDate.Text = "Show relative date instead of full date";
+            chkShowRelativeDate.UseVisualStyleBackColor = true;
             // 
             // chkShowRepoCurrentBranch
             // 
-            this.chkShowRepoCurrentBranch.AutoSize = true;
-            this.tlpnlGeneral.SetColumnSpan(this.chkShowRepoCurrentBranch, 2);
-            this.chkShowRepoCurrentBranch.Dock = System.Windows.Forms.DockStyle.Fill;
-            this.chkShowRepoCurrentBranch.Location = new System.Drawing.Point(3, 26);
-            this.chkShowRepoCurrentBranch.Name = "chkShowRepoCurrentBranch";
-            this.chkShowRepoCurrentBranch.Size = new System.Drawing.Size(448, 17);
-            this.chkShowRepoCurrentBranch.TabIndex = 5;
-            this.chkShowRepoCurrentBranch.Text = "Show current branch names in the dashboard and the recent repositories dropdown menu";
-            this.chkShowRepoCurrentBranch.UseVisualStyleBackColor = true;
+            chkShowRepoCurrentBranch.AutoSize = true;
+            tlpnlGeneral.SetColumnSpan(chkShowRepoCurrentBranch, 2);
+            chkShowRepoCurrentBranch.Dock = DockStyle.Fill;
+            chkShowRepoCurrentBranch.Location = new Point(3, 28);
+            chkShowRepoCurrentBranch.Name = "chkShowRepoCurrentBranch";
+            chkShowRepoCurrentBranch.Size = new Size(501, 19);
+            chkShowRepoCurrentBranch.TabIndex = 1;
```

Comment:
Remove TabIndex instead?
It will show up the next time Designer is used though.

Reply:
> Remove TabIndex instead?

I tried this first. It does not work.
I guess then the sequence of addition to parent controls is used.

Reply:
The designer always emits the `TabIndex` property. I often manually delete the entries, so that a dialog gets the "natural" tab order.